.gitlab-ci.yml

```diff
@@ -190,7 +190,7 @@ integration: # merging all artifacts so multiple deploy jobs can build same repo
     - linux
   only:
     - gl-ci-upgrade
-    - master
+    - roll
```

Comment:
this was only added for development to render manual into html https://jangorecki.gitlab.io/data.table/library/data.table/html/roll.html
to be removed before merge